### PR TITLE
Add Katana Engine

### PR DESCRIPTION
**Pull request by MichaelHinrichs**
### SteamDB app page links to a few games using this
https://steamdb.info/app/1072420
https://steamdb.info/app/1938010
https://steamdb.info/app/1382330

### Brief explanation of the change
Find Katana engine file types.

### CODE REVIEW

**Comment by SteamTracker** [comment]
After running changed rules in this pull request *(this bot only tests rules.ini changes)*:

Changed technologies: **Engine.Katana**

These apps now match: **(22)** [449800](https://steamdb.info/app/449800/depots/) [485510](https://steamdb.info/app/485510/depots/) [601050](https://steamdb.info/app/601050/depots/) [628890](https://steamdb.info/app/628890/depots/) [730310](https://steamdb.info/app/730310/depots/) [1019100](https://steamdb.info/app/1019100/depots/) [1019870](https://steamdb.info/app/1019870/depots/) [1045620](https://steamdb.info/app/1045620/depots/) [1121560](https://steamdb.info/app/1121560/depots/) [1257290](https://steamdb.info/app/1257290/depots/) [1325200](https://steamdb.info/app/1325200/depots/) [1341200](https://steamdb.info/app/1341200/depots/) [1423600](https://steamdb.info/app/1423600/depots/) [1424800](https://steamdb.info/app/1424800/depots/) [1502970](https://steamdb.info/app/1502970/depots/) [1502980](https://steamdb.info/app/1502980/depots/) [1502990](https://steamdb.info/app/1502990/depots/) [1582390](https://steamdb.info/app/1582390/depots/) [1621310](https://steamdb.info/app/1621310/depots/) [1999770](https://steamdb.info/app/1999770/depots/) [2130460](https://steamdb.info/app/2130460/depots/) [2138090](https://steamdb.info/app/2138090/depots/)

Please confirm that all these changed apps are correct.

File: rules.ini [hunk 1]
```diff
@@ -122,6 +122,7 @@ Infernal_Engine = (?:^|/)(?:COMMON|STARTUP)\.POD$
 Irrlicht = (?:^|/)Irrlicht\.dll$
 IW_Engine[] = \.iwd$
 IW_Engine[] = (?:common(?:_zm|_mp)?|code_post_gfx)\.ff$
+Katana = \.ktsl2(?:as|gc|st)bin$
```
Comment (Lifeismana) [inline rules.ini:125]:
```suggestion
Katana = \.ktsl2(?:as|st)bin$
```

File: tests/types/Engine.Katana.txt
Comment (Lifeismana) [inline tests/types/Engine.Katana.txt:1]: add some real strings

**Comment by Lifeismana** [comment]
merged in 566eb5d45d441320d9e538a743a8f23515b0ab59

**Pull request closed without merging**